website/docs/automation/gitops/argocd/cluster_bootstrap.md

```diff
@@ -18,22 +18,20 @@ The Git repository in AWS CodeCommit has already been created for you.
 If you want to use your own GitHub private repository you could re define
 
 ```
-GITOPS_REPO_URL=https://github.com/username/reponame
-echo $GITOPS_REPO_URL > ~/environment/argocd_repo_url
+export GITOPS_REPO_URL_ARGOCD=https://github.com/username/reponame
 ```
 
 and use [those instructions](https://argo-cd.readthedocs.io/en/stable/user-guide/private-repositories/) to create an Argo CD secret to give access to the Git repository from Argo CD
 :::
 
 ```bash
-$ GITOPS_REPO_URL=ssh://${GITOPS_IAM_SSH_KEY_ID}@git-codecommit.${AWS_DEFAULT_REGION}.amazonaws.com/v1/repos/${EKS_CLUSTER_NAME}-argocd
-$ echo $GITOPS_REPO_URL > ~/environment/argocd_repo_url
+$ export GITOPS_REPO_URL_ARGOCD=ssh://${GITOPS_IAM_SSH_KEY_ID}@git-codecommit.${AWS_DEFAULT_REGION}.amazonaws.com/v1/repos/${EKS_CLUSTER_NAME}-argocd
```

Comment:
Remove this export is not need it, since is in the shell env now

Reply:
Done.